### PR TITLE
Can now create and view flashcards

### PR DESCRIPTION
**Pull request by elGatoMantocko**

### CODE REVIEW

**Comment by elGatoMantocko** [comment]
This still needs some CSS and jQuery magic, but that shouldn't be too difficult. I will have this merged by the time I go to bed tonight.

**Comment by elGatoMantocko** [comment]
I realize we may set it up later so that each card ID is stored into a deck collection, but I think for now it may be easier to store the course ID into each card and do a select on the cards based on that course ID. This has yet to be implemented though.

At the current state, each course will display every card.

**Comment by cjsauer** [comment]
Copied from hipchat for reference: 

> Commit c18a801 is a lie
> I have it now so that each flashcard has a courseID field
> Courses now only display flashcards that actually belong to them, and upon creation, they auto-fill the courseID field depending on what route you're on.
> There might be a better way to do it. I have an AutoForm hook setup in FlashcardNew.js right now to append the courseID before insertion
> I think that way it will be impossible for a client to try any funny business
> Not that that's a big deal anyway haha
> @elliott pull that down and give it a shot
> I cleaned up a bit of your template and helper code. Mainly, I moved the flashcard templates around so that there's no code duplication. The form and show templates now both utilize a common "parent" template

File: client/views/flashcards/FlashcardNew.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template name="FlashcardNew">
+<div class="input-group input-group-sm text-center">
+  {{#autoForm id="new-card" type="insert" collection="Flashcards"}}
```
Comment (cjsauer) [inline client/views/flashcards/FlashcardNew.html:3]: In the commits before this, you had a <form> tag before the {{autofrom}} helper. This created two <form> tags in the html after rendering. That's why some of the styles are a bit wonky now.

**Comment by elGatoMantocko** [comment]
I think the only thing that is left to be done is a conditional that changes the placeholder text on the afFieldInput in FlashcardNew.html.

If the index is zero on the afArrayField loop, then the placeholder should say 'answer' else it should say 'response' or something like that.

File: client/views/flashcards/Flashcard.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  }
+});
+
+Template.Flashcard.helpers({
+  indexIsZero: function() {
+    return this.index == 0
+  }
+});
+
+UI.registerHelper('addIndex', function (all) {
```
Comment (cjsauer) [inline client/views/flashcards/Flashcard.js:14]: What is this for? I can't find anything in the Meteor docs for this. When I removed it everything is still working...

Comment (elGatoMantocko) [inline client/views/flashcards/Flashcard.js:14]: Yeah its safe to remove, it was an original idea I had that got merged into my work. I forgot to remove it.

**Pull request merged**